### PR TITLE
Create symlinks for merged cgroups

### PR DESCRIPTION
**Pull request by LK4D4**
This allows software be not aware about existence of merged cgroups.

ping @mrunalp

### CODE REVIEW

**Comment by mrunalp** [comment]
Needs rebase now :)

**Comment by LK4D4** [comment]
haha

**Comment by LK4D4** [comment]
@mrunalp Now it's even more cleaner

**Comment by mrunalp** [comment]
LGTM

File: libcontainer/rootfs_linux.go [hunk 1]
```diff
@@ -188,6 +195,29 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
...
+		if err != nil {
+			return err
+		}
+		if err := os.Chdir(filepath.Join(rootfs, m.Destination)); err != nil {
+			return err
+		}
+		for _, mc := range merged {
+			for _, ss := range strings.Split(mc, ",") {
+				if err := os.Symlink(mc, ss); err != nil {
+					// if cgroup already exist, then okay(it could be created before)
```
Comment (mrunalp) [inline libcontainer/rootfs_linux.go]: s/exist/exists
"it could have been created before"

**Comment by LK4D4** [comment]
ping @avagin @rjnagal

**Comment by rjnagal** [comment]
LGTM

**Pull request merged**